packages/pcd/eddsa-pcd/src/util/util.ts

```diff
@@ -1,6 +1,21 @@
-import { getRandomValues, toHexString } from "@pcd/util";
+import { fromHexString, getRandomValues, toHexString } from "@pcd/util";
+import { Point } from "@zk-kit/baby-jubjub";
+import { packPublicKey, unpackPublicKey } from "@zk-kit/eddsa-poseidon";
+import {
+  BigNumber,
+  bigIntToHexadecimal,
+  hexadecimalToBigInt,
+  leBigIntToBuffer,
+  leBufferToBigInt
+} from "@zk-kit/utils";
 import { EdDSAPublicKey } from "../EdDSAPCD";
 
+/**
```

Comment:
I think we can probably make Base64 and hex both be supported in future if we want to, since they can be distinguished by their length.